src/visitor.h

```diff
@@ -559,6 +560,7 @@ class Visitor
     virtual void visit(CondExp *e) { visit((BinExp *)e); }
     virtual void visit(DefaultInitExp *e) { visit((Expression *)e); }
     virtual void visit(FileInitExp *e) { visit((DefaultInitExp *)e); }
+    //virtual void visit(FileFullPathInitExp *e) { visit((DefaultInitExp *)e); }
```

Comment:
Caused the build to fail...not sure why.  Got an assert or some nasty error. Not sure how useful it is, but since everything else had a visitor callback I thought I would leave it as a comment for someone else to fix if they ever needed to specifically visit the FileFullPathInitExp class.